app/presenters/hyrax/file_set_presenter.rb

```diff
@@ -0,0 +1,107 @@
+module Hyrax
+  class FileSetPresenter
+    include ModelProxy
+    include PresentsAttributes
+    include CharacterizationBehavior
+    include WithEvents
+    include DisplaysImage
+
+    attr_accessor :solr_document, :current_ability, :request
+
+    # @param [SolrDocument] solr_document
+    # @param [Ability] current_ability
+    # @param [ActionDispatch::Request] request the http request context
+    def initialize(solr_document, current_ability, request = nil)
+      @solr_document = solr_document
+      @current_ability = current_ability
+      @request = request
+    end
+
+    # CurationConcern methods
+    delegate :stringify_keys, :human_readable_type, :collection?, :image?, :video?,
+             :audio?, :pdf?, :office_document?, :representative_id, :to_s, to: :solr_document
+
+    # Methods used by blacklight helpers
+    delegate :has?, :first, :fetch, to: :solr_document
+
+    # Metadata Methods
+    delegate :title, :label, :description, :creator, :contributor, :subject,
+             :publisher, :language, :date_uploaded,
+             :embargo_release_date, :lease_expiration_date,
+             :depositor, :keyword, :title_or_label, :keyword,
+             :date_created, :date_modified, :itemtype,
+             to: :solr_document
+
+    # Exif TechnicalMetadata
```

Comment:
if this is the only thing you are overriding might be a good idea to just put this in `class_eval` in an initializer so we don't have to track Hyrax changes to the rest of the code in the `FIleSetPresenter`?